templates/settings.py.erb

```diff
@@ -27,3 +27,17 @@ WORKING_DIRECTORY = "<%= scope['pulpcore::cache_dir'] %>"
 <% if scope['pulpcore::remote_user_environ_name'] -%>
 REMOTE_USER_ENVIRON_NAME = '<%= scope['pulpcore::remote_user_environ_name'] %>'
 <% end -%>
+AUTHENTICATION_BACKENDS = ['pulpcore.app.authentication.PulpNoCreateRemoteUserBackend']
+
+REST_FRAMEWORK__DEFAULT_AUTHENTICATION_CLASSES = (
+    'rest_framework.authentication.SessionAuthentication',
+    'pulpcore.app.authentication.PulpRemoteUserAuthentication'
+)
+
+<%# This setting whitelists paths that can be used for repository sync with file protocol. Katello uses the path /var/lib/pulp/sync_imports/ to run tests -%>
+<% allowed_paths = [] -%>
```

Comment:
Why is this needed? If anything you can use `ALLOWED_IMPORT_PATHS = <%= Array.new(scope['pulpcore::allowed_import_path']) %>` but I think it should not be needed since it's already an array.

Reply:
That is correct..Updated..This works `scope['pulpcore::allowed_import_path']` .. :+1: